addons/filesystem/functions/fnc_chown.sqf

```diff
@@ -0,0 +1,33 @@
+params['_pntr', '_filesystem', '_target', '_user', '_owner', ['_recursive', false]];
```

Comment:
If there is a `chown` filesystem command, we should allow this command as an os command, too.

Reply:
I will do this, when getOpts is merged